conan/cli/command.py

```diff
@@ -154,9 +168,10 @@ def add_subcommand(self, subcommand):
     def run_cli(self, conan_api, *args):
         parser = ConanArgumentParser(conan_api, description=self._doc,
                                      prog="conan {}".format(self._name),
-                                     formatter_class=SmartFormatter)
-        self._init_log_levels(parser)
-        self._init_formatters(parser)
+                                     formatter_class=SmartFormatter,
+                                     add_help=False,
+                                     command=self)
+        parser.suggest_on_error = True
```

Comment:
Python 3.14 feature, where arguments that have a `choice` now give you suggestions for typos in the value if possible.

Done this way so older Python versions do not break when using this attribute, as per https://docs.python.org/3/library/argparse.html#suggest-on-error